floris/tools/optimization/scipy/yaw_wind_rose.py

```diff
@@ -404,33 +494,35 @@ def reinitialize_opt_wind_rose(
             self.opt_method = opt_method
         if opt_options is not None:
             self.opt_options = opt_options
+        self.x_baseline = [
+                turbine.yaw_angle
+                for turbine in self.fi.floris.farm.turbine_map.turbines
+        ]
         if x0 is not None:
             self.x0 = x0
+            if not all(np.array(x0) == np.array(self.x_baseline)):
+                print("WARNING: Baseline yaw angle in FLORIS differ from initial optimization conditions (self.x0)")
```

Comment:
Also don't think a warning is necessary here.

Reply:
Changed this to an INFO statement. It might not be necessary indeed, but since I expect it to be an uncommon usage of the code, I want to print this statement to ensure the user is doing what he/she is intending.